cmd/server/customer_search_scope_test.go

```diff
@@ -57,6 +58,19 @@ func (scope *Scope) CreateCustomers(count int){
 		if err := scope.customerRepo.createCustomer(cust); err != nil {
```

Comment:
Should this call `scope.CreateCustomer(..)`?

Reply:
It totally should. I'll make that change in an upcoming PR.